pkg/security/podsecuritypolicy/group/mustrunas_test.go

```diff
@@ -171,5 +177,8 @@ func TestValidate(t *testing.T) {
 		if !v.pass && len(errs) == 0 {
 			t.Errorf("expected no errors for %s but got: %v", k, errs)
 		}
+		if !v.pass && !strings.Contains(errs[0].Error(), v.expectedError) {
+			t.Errorf("expected error for %s: %v, but got: %v", k, v.expectedError, errs[0].Error())
```

Comment:
nit: instead of `errs[0].Error()` you can use `errs[0]` and it will work because of `%v`.